code/modules/mob/living/carbon/xenomorph/abilities.dm

```diff
@@ -39,6 +39,93 @@
 
 	return succeed_activate()
 
+/datum/action/xeno_action/activable/headbite
+	name = "Headbite"
+	action_icon_state = "headbite"
+	mechanics_text = "Permanently kill a target."
+	use_state_flags = XACT_USE_STAGGERED|XACT_USE_FORTIFIED|XACT_USE_CRESTED
+	keybind_signal = COMSIG_XENOABILITY_HEADBITE
+	plasma_cost = 100
+
+/datum/action/xeno_action/activable/headbite/can_use_ability(atom/A, silent = FALSE, override_flags)
+	. = ..()
+	var/mob/living/carbon/victim = A
+	if(!.)
+		return FALSE
+	if(!owner.Adjacent(A))
+		return FALSE
+	if(locate(/obj/item/alien_embryo) in victim) //Maybe they ate it??
+		var/mob/living/carbon/human/H = victim
+		if(CHECK_BITFIELD(H.status_flags, XENO_HOST))
+			if(victim.stat != DEAD) //Not dead yet.
+				if(!silent)
+					to_chat(owner, "<span class='xenowarning'>The host and child are still alive!</span>")
+				return FALSE
```

Comment:
```suggestion
		if(CHECK_BITFIELD(H.status_flags, XENO_HOST))
			if(victim.stat != DEAD) //Not dead yet.
				if(!silent)
					to_chat(owner, "<span class='xenowarning'>The host and child are still alive!</span>")
				return FALSE
```
try using && here to avoid indenting so much